shared/src/business/useCases/migration/parseLegacyDocumentsInteractor.js

```diff
@@ -75,10 +76,17 @@ exports.parseLegacyDocumentsInteractor = async ({
 
   foundDocketEntry.documentContentsId = documentContentsId;
 
+  const validatedDocketEntry = new DocketEntry(foundDocketEntry, {
+    applicationContext,
+  }).validate();
```

Comment:
Oops, did I cause a bug before? I didn’t see how this could not pass validation (since we’re just adding the contents) but please educate me if I made a mistake! 🐑

Reply:
I don't think you made a mistake, but I don't think you had a good way to test it. And so I learned something to trying to test this.

The persistence gateway was throwing an error that the DocketEntry hadn't been validated, and so it put the brakes on the process before updating the DB. All I had to do was validate it first, and then the save was successful. In looking closer at that validation process, I'm a little curious about [what other effects that has on the entity](https://github.com/ustaxcourt/ef-cms/blob/staging/shared/src/business/entities/DocketEntry.js#L133).